front/src/components/Messages.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react'
 import styled from 'styled-components'
 import colors from '../colors'
+import './ToolTip.css'
 
 const MessageDiv = styled.div`
   // Make it easy to visually differentiate messages
   padding-bottom: 7px;
 `
 
 const Message = (props) => {
+  console.log(props.message.playerID)
   return (
-    <MessageDiv>
+    <MessageDiv className="tooltip">
       {/* IMPORTANT SPACE --> */}
-      <b>&lt;{props.message.sender}&gt;</b> <span>{props.message.content}</span>
+      <b>&lt;{props.message.sender}&gt;</b> {props.message.content}
+      <span className={props.message.playerID ? 'tooltiptext' : ''}>{props.message.playerID}</span>
```

Comment:
Alright. This could be done better if you use the styled-components library. The basics of styled-components are very simple. See below.
```js
// `styled` contains a series of functions representing html elements (`styled.div`, `styled.span` etc).
// The element functions takes a template string containing css and returns a component with the styles aplied
// For example 
const RedDiv = styled.div`
color: red;
`
```

In this case, instead of adding the `tooltip` class, you would add the styling directly to the MessageDiv styling template. If you need some common styling between two components just put it in a constant string and use a ${} placeholder to add it to both of them.

```js
const MessageDiv = styled.div`
  // Make it easy to visually differentiate messages
  padding-bottom: 7px;
  // Add your styling here
  // ...
  // Import common styling
  ${commonStyling}
````

```js
const ToolTip = styled.span`
  // Add your styling here
  // ...
  // Import common styling
  ${commonStyling}
```

Reply:
Yeah, I will try that in the morning

Reply:
Great!